games/default/files/default/nodes.lua

```diff
@@ -1710,21 +1723,6 @@ minetest.register_node("default:grass", {
     wield_image = "default_tallgrass.png",
     walkable = false,
     buildable_to = true,
-        --drop = {
```

Comment:
Please, back this!
I uncomment this code, when farming mod will be done.

Reply:
Done.